mobile/lib/screens/kya/kya_title_page.dart

```diff
@@ -69,7 +69,10 @@ class _KyaTitlePageState extends State<KyaTitlePage> {
                         context,
                         MaterialPageRoute(
                           builder: (context) {
-                            return KyaLessonsPage(widget.kya);
+                            return KyaLessonsPage(
+                              kya: widget.kya,
+                            );
+                            // return;
```

Comment:
minimise comments as much as possible. You can just use PR review comments to guide your reviewers. Thanks!